dist/StorageManager.js

```diff
@@ -0,0 +1,36 @@
+class StorageManager{
+  constructor(){
+    this.savedTeams = JSON.parse(localStorage.getItem('teams')) || []
+  }
+  saveTeam = (teamName, teamData, renderCallback) => {
+    if (teamName.length < 1) {
+      renderCallback(this.getAllTeams(),'Please Enter Team Name')
+      return
+    }
+    if (teamData.length < 5) {
+      renderCallback(this.getAllTeams(),'Please Save Team of 5')
```

Comment:
You should try to make separation of concerns as much as you can - so this StorgaeManager should be all about the local storage, without any rendering